search-flow.yml

```
@@ -0,0 +1,41 @@
+jtype: Flow
+version: '1'
+with:
+  protocol: 'http'
+  cors: True
+  port_expose: $JINA_PORT
+executors:
+  - name: text_encoder
+    uses: jinahub+docker://AudioCLIPTextEncoder/v0.3
+    uses_with:
+      traversal_paths: ['r',]
+    volumes: $MODEL_MOUNT_CACHE
+    needs: gateway
+  - name: image_indexer
+    uses: jinahub://SimpleIndexer/v0.4
+    uses_with:
+      match_args:
+        limit: $TOP_K
+        traversal_ldarray: ['r',]
+        traversal_rdarray: ['c',]
+    uses_metas:
+      workspace: $JINA_WORKSPACE/image_indexer
+    volumes: $WORKSPACE_MOUNT
+    needs: ['text_encoder']
+  - name: audio_indexer
+    uses: jinahub://SimpleIndexer/v0.4
+    uses_with:
+      match_args:
+        limit: $TOP_K
+        traversal_ldarray: ['r',]
+        traversal_rdarray: ['c',]
+    uses_metas:
+      workspace: $JINA_WORKSPACE/audio_indexer
+    volumes: $WORKSPACE_MOUNT
+    needs: ['image_indexer']
```

The simpleindexer will override the matches. Plus, we need to `MatchMerger` to merge the matches from the two pathways.

Reference:
https://github.com/jina-ai/jina/blob/master/jina/types/arrays/neural_ops.py#L124
```suggestion
    needs: ['text_encoder']
```